plugin/hpsa/hpsa.py

```diff
@@ -231,6 +231,17 @@ def _disk_status_of(hp_disk, flag_free):
 
     return disk_status
 
+def _disk_link_type_of(hp_disk):
+    disk_interface = hp_disk['Interface Type']
+    if disk_interface == 'SATA':
+        return Disk.LINK_TYPE_ATA
+    elif disk_interface == 'Solid State SATA':
+        return Disk.LINK_TYPE_ATA
+    elif disk_interface == 'SAS':
+        return Disk.LINK_TYPE_SAS
+
```

Comment:
@joehandzik I don't have SAS SSD connected to our hpsa card and failed to find any document indicate possible values of 'Interface Type'. Above is what I find in our lab.
If that is the property value for SAS SSD, I could add it. Could you confirm that with me or even better provide output of `hpssacli ctrl all show config detail` on controller with SAS SSD?
Thank you.